packages/dd-trace/src/appsec/iast/taint-tracking/rewriter.js

```diff
@@ -157,7 +157,7 @@ function shimPrepareStackTrace () {
     return
   }
   const pstDescriptor = Object.getOwnPropertyDescriptor(global.Error, 'prepareStackTrace')
-  if (pstDescriptor?.configurable || pstDescriptor?.writable) {
+  if (!pstDescriptor || pstDescriptor.configurable || pstDescriptor.writable) {
```

Comment:
In node 18, there is no `Error.prepareStackTrace` defined by default, so the descriptor is undefined, and in that case the `prepareStackTrace` should be overridden.